src/qt/veil.cpp

```diff
@@ -609,6 +608,9 @@ int main(int argc, char *argv[])
     QSslConfiguration::setDefaultConfiguration(sslconf);
 #endif
 
+    BitcoinApplication app(*node, argc, argv);
+
```

Comment:
Remove extra white space here, other than that all good.

Reply:
done ✍✍